### DISCUSSION TITLE
问一下开了弹幕翻译,但是偶尔会出现,没有被翻译的弹幕的问题

### DISCUSSION THREAD

**Discussion by flywolf53** [xfgryujk/blivechat#204] [提问]
是翻译api的问题吗,还是软件设置什么的问题

**Comment by xfgryujk** [comment]
弹幕速度太快了，或者请求失败了就会这样

**Comment by flywolf53** [comment]
> 弹幕速度太快了，或者请求失败了就会这样

请求失败不太理解是哪方面的问题,那我这边没有什么解决的办法了是么

**Comment by xfgryujk** [comment]
> > 弹幕速度太快了，或者请求失败了就会这样
> 
> 请求失败不太理解是哪方面的问题,那我这边没有什么解决的办法了是么

你先看后台日志有没有请求失败的，如果配了官方接口失败的概率应该不高
